toolkit/tools/imagegen/configuration/parse_partition.go

```diff
@@ -89,7 +89,17 @@ func processDisk(inputDiskValue string) (err error) {
 			disks = append(disks, Disk{})
 		}
 
-		disks[latestDiskIndex].PartitionTableType = PartitionTableTypeGpt
+		// Read from /proc/cmdline to determine if the user is intending for a GPT partition
+		partitionTableType, err := GetKernelCmdLineValue("--gpt", 6)
```

Comment:
I'm curious if the user shouldn't define this inside the config file instead. How are the kernel arguments set, that we don't see them in our config but are already available to the installer at this stage and represent the final setting the user is interested in?

Reply:
Its a unique feature of the iso/pxe scenario. The installer's kernel is launched with the cmdline arguments set.

Reply:
Are you saying that these arguments the same as the ones the installed kernel is going to run with?

Reply:
No, the PXE boot path uses the kernel that is included in the ISO. That kernel will be given some cmdline args which the installer tools can read from.

Reply:
Thanks, Daniel! I talked a little bit more with Chris to get a better understanding of what's happening here and am replacing this comment with two smaller ones.;)